scripts/generate-assets.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+const path = require('path');
+const fs = require('fs');
+const { promisify } = require('util');
+const { parse } = require('@stoplight/yaml');
+const { httpAndFileResolver } = require('../dist/resolvers/http-and-file');
+
+const readFileAsync = promisify(fs.readFile);
+const writeFileAsync = promisify(fs.writeFile);
+const readdirAsync = promisify(fs.readdir);
+const statAsync = promisify(fs.stat);
+
+const baseDir = path.join(__dirname, '../rulesets/assets/');
+
+if (!fs.existsSync(baseDir)) {
+  fs.mkdirSync(baseDir);
+}
+
+const target = path.join(baseDir, `assets.json`);
+const assets = {};
+
+(async () => {
```

Comment:
Is this change needed if we are doing #561 ?

Reply:
Yeah, it's still going to be needed. It's mostly for Studio to use and we don't expose it anywhere in the docs, etc.
I'll leave a note somewhere in the code on what it actually does and how it works, as right now it's not obvious, and we will forget about why we need it.
I see Windows build is failing, so need to take a look what's broken.

Reply:
We started a 2 week free trial Wednesday afternoon

Reply:
@P0lip ok, fair enough on the still needed. If you cannot get windows build working with Azure then maybe try out the CircleCI windows build, however that works.

Reply:
@philsturgeon I got Windows build working, but would love to try out Windows on CircleCI. Azure pipelines is tad slow.

Reply:
Another time! :)